test/test_nn.py

```diff
@@ -5760,6 +5760,38 @@ def test_RNN_cell_forward_hidden_size(self):
                      nn.GRUCell(*cell_shared_param)):
             self.assertRaises(Exception, lambda: cell(input, hx))
 
+    def test_GroupNorm_numeric(self):
```

Comment:
can you move this test into device-generic part of the test suite (TestNNDeviceType at line ~11568) so that you'd have device argument to handle cuda testing?
How long does the cpu test take? Should it be decorated with @slowTest?

Reply:
Done.